bundles/org.openhab.binding.meater/README.md

```diff
@@ -0,0 +1,56 @@
+# Meater Binding
+
+This is an openHAB binding for the Meater probe, by Apption Labs.
+
+This binding uses the Meater cloud REST API.
+
+![Meater+ Probe](doc/meater-plus-side.png)
+
+## Supported Things
+
+This binding supports the following thing types:
+
+- meaterapi: Bridge - Communicates with the Meater cloud REST API.
+
+
+- meaterprobe: The Meater probe - Only support for cloud connected Meater probes (Meater+) 
+
+## Discovery
+
+The preferred way of adding Meater probe(s) since the probe IDs are not easily found.
+
+**NOTE**: You need to have your Meater probe(s) connected to the cloud and the Meater app running before you start the discovery.
+
+After the configuration of the Bridge, your Meater probe(s) will be automatically discovered and placed as a thing(s) in the inbox.
+
+
+## Thing Configuration
+
+#### Bridge
+
+| Parameter | Description                                                  | Type   | Default  | Required | 
+|-----------|--------------------------------------------------------------|--------|----------|----------|
+| email     | The email used to connect to your Meater cloud account       | String | NA       | yes      |
+| password  | The password used to connect to your Meater cloud account    | String | NA       | yes      |
+| refresh   | Specifies the refresh interval in second                     | Number | 30       | yes      |
```

Comment:
It is not required as a default value is defined.

Reply:
Fixed!